app/models/criterion.rb

```diff
@@ -101,62 +103,51 @@ def self.unassign_tas(criterion_ta_ids, criterion_ids_by_type, assignment)
   # an assignment with ID +assignment_id+.
   def self.update_assigned_groups_counts(assignment, criterion_ids_by_type = nil)
     # Sanitize the IDs in the input.
-    if criterion_ids_by_type.nil? or criterion_ids_by_type['RubricCriterion'].nil?
```

Comment:
These changes should not be in this pull request. You should revert them and make them in a separate branch.